rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -639,3 +639,42 @@ def _get_address_set(self, set_name):
 
         ovn_nbctl = self._get_ovn_controller(self.install_method)
         return ovn_nbctl.get("Address_Set", set_name, 'addresses')
+
+    def runCmd(self, ssh, cmd=""):
+        ssh.enable_batch_mode(False)
+
+        stdout = StringIO()
+        ssh.run(cmd, stdout=stdout)
+        return stdout.getvalue().rstrip()
+
+    def runFarmCmd(self, sandbox, cmd="", pid_opt="", background_opt=False):
+        LOG.info("running %s on %s" % (cmd, sandbox["name"]))
+
+        ssh = self.farm_clients(sandbox["farm"], "ovs-ssh")
+        ssh.set_sandbox(sandbox["name"], self.install_method,
+                        sandbox["host_container"])
+
+        if pid_opt:
+            pid = self.runCmd(ssh, "pidof -s ovn-controller")
```

Comment:
What if we want to run a command like:
 ```
perf record -p `pidof -s ovs-vswitchd`
```

I think we need another `pid_proc_name` argument, or something like that.

Reply:
@dceara ack, fine I will update it